package.json

```diff
@@ -151,5 +150,10 @@
     "vue": "^3.2.31",
     "vue-i18n": "v9.1.9",
     "vue-router": "^4.0.14"
+  },
+  "pnpm": {
+    "overrides": {
+      "@parcel/watcher": "link:tools/prebuilds/@parcel/watcher"
```

Comment:
@apolkingg8 This is a hack to include a prebuild for `@parcel/watcher`, which is used by Nx (or Nest?) to reload the server. The reason is that it doesn't include a M1 prebuild by default, so I include it here manually to avoid g++/make/python3 dependencies in Docker.